main.py

```diff
@@ -66,6 +67,7 @@ def __call__(self, request, response, exception):
       RedirectRoute(r'/logout', AccountLogout, 'account-logout', strict_slash=True),
       RedirectRoute(r'/match/<match_key>', MatchDetail, 'match-detail', strict_slash=True),
       RedirectRoute(r'/notifications/broadcast', UserNotificationBroadcast, 'notification-broadcast', strict_slash=True),
+      RedirectRoute(r'/notifications/test/([0-9]+)', TestNotificationController),
```

Comment:
That's what I get for pushing and deploying to test instance at the same time :P